src/secrets/components/SecretContextMenu.tsx

```diff
@@ -42,19 +50,35 @@ const SecretContextMenu: FC<Props> = ({secret}) => {
     }
   }
 
+  // Note to self: add a class on the context element to push everything left about
+  // 20 pixels because there's no padding on the rightmost button for reasons
```

Comment:
🤷🏽

Reply:
I did take care of this, just forgot to remove the comment.  Removed it now.